lib/classes/CLI.js

```diff
@@ -220,6 +220,39 @@ class CLI {
     console.log('');
   }
 
+  printError(e) {
+    this.asciiGreeting();
```

Comment:
Do we really need the greeting? I like it in some scenarios but it's very large and distracts from the actual output IMO.